e2e-tests/cypress/e2e/journeys/insurance/account/sign-in/validation/account-sign-in-validation.spec.js

```diff
@@ -0,0 +1,107 @@
+import partials from '../../../../../partials';
+import { submitButton } from '../../../../../pages/shared';
+import accountFormFields from '../../../../../partials/insurance/accountFormFields';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { INSURANCE_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance';
+import { INSURANCE_ROUTES as ROUTES } from '../../../../../../../constants/routes/insurance';
+import account from '../../../../../../fixtures/account';
+
+const {
+  START,
+  ACCOUNT: {
+    SIGN_IN: { ROOT: SIGN_IN_ROOT },
+  },
+} = ROUTES;
+
+const {
+  ACCOUNT: { EMAIL, PASSWORD },
+} = INSURANCE_FIELD_IDS;
+
+const {
+  INSURANCE: { ACCOUNT: { SIGN_IN: SIGN_IN_ERROR_MESSAGES } },
+} = ERROR_MESSAGES;
+
+const assertErrorMessages = () => {
```

Comment:
can we use my cy command for tests here?

Reply:
Great shout, forget about this new command - have updated 👍